src/objc.ts

```diff
@@ -370,6 +370,7 @@ export interface Class {
   baseClassName:string;
   classMethods:Method[];
   comments:Comment[];
+  generics:string[];
```

Comment:
This is also a place where I would consider using a Maybe type. There are a bunch of places where you're passing `[]` around which seems like the API is underspecified. Not a huge deal, and I know that we're doing that in other places too, it just feels more noticeable here I guess